cornucopia/src/error.rs

```diff
@@ -11,9 +11,7 @@ pub enum Error {
     ValidateQueries(#[from] crate::validation::error::Error),
     Container(#[from] crate::container::error::Error),
     PrepareQueries(#[from] crate::prepare_queries::error::Error),
-    NewMigration(#[from] NewMigrationError),
-    RunMigration(#[from] crate::run_migrations::error::Error),
-    ReadMigration(#[from] crate::read_migrations::error::Error),
+    ReadMigration(#[from] crate::read_schema::error::Error),
```

Comment:
`ReadSchema` instead of `ReadMigration` (c.f. my comment on the PR thread about the `read_schema` module name)